### PR TITLE
Merge voice-change-o-matic repo with history

### PR DESCRIPTION
**Pull request by bsmth**
Moving in the `voice-change-o-matic` repo and preserving history as described in https://hacks.mozilla.org/2022/08/merging-two-github-repositories-without-losing-commit-history/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (schalkneethling) [inline .gitignore:1]: We can probably delete this one.

File: voice-change-o-matic/LICENSE [hunk 2]
```diff
@@ -0,0 +1,116 @@
+CC0 1.0 Universal
```
Comment (schalkneethling) [inline voice-change-o-matic/LICENSE:1]: I believe this is the same license that the repository as a whole uses so, we can remove it here.

**Review by schalkneethling — COMMENTED** [review]
Thanks, @bsmth 🙌 - Two small things.

File: .gitignore (hunk 1 above)
Comment (bsmth) [reply .gitignore:1]: okie doke

File: voice-change-o-matic/LICENSE (hunk 2 above)
Comment (bsmth) [reply voice-change-o-matic/LICENSE:1]: good 👍🏻

Comment (bsmth) [reply voice-change-o-matic/LICENSE]: done in https://github.com/mdn/webaudio-examples/pull/92/commits/053ab323bd89e51dfecaac43e8157dcec784b00a

File: .gitignore (hunk 1 above)
Comment (bsmth) [reply .gitignore]: done in https://github.com/mdn/webaudio-examples/pull/92/commits/053ab323bd89e51dfecaac43e8157dcec784b00a

**Review by schalkneethling — APPROVED** [review]
🎉 🌮

**Pull request merged**